src/tito/tagger/main.py

```diff
@@ -478,9 +478,16 @@ def _update_package_metadata(self, new_version):
         run_command("git add %s" % os.path.join(self.full_project_dir,
             self.spec_file_name))
 
-        run_command('git commit -m "Automatic commit of package ' +
-                '[%s] %s [%s]."' % (self.project_name, self.release_type(),
-                    new_version_w_suffix))
+        fmt = ('Automatic commit of package '
+               '[%(name)s] %(release_type)s [%(version)s].')
+        if self.config.has_option(BUILDCONFIG_SECTION, "commit_message_format"):
```

Comment:
Could I just get a rename on this property to "tag_commit_msg_format", I know that's probably the only git commit we do but just incase for the future. Everything else looks good to me.